docs/api/abi.rst

```diff
@@ -3,7 +3,7 @@ Abi
 
 Module containing representation of contract abi and parser for creating it from parsed json.
 
-.. py:module:: starknet_py.abi
+.. py:module:: starknet_py.abi.v2
```

Comment:
Could we also include `AbiParser` and `Abi` documentation for `v0` and `v1` at the bottom of this file?